fdbserver/storageserver.actor.cpp

```diff
@@ -512,7 +512,7 @@ struct StorageServer {
 					.detail("TagCost", busiestTagCount)
 					.detail("TotalSampledCost", intervalTotalSampledCount)
 					.detail("Reported", previousBusiestTag.present())
-					.trackLatest(id.toString() + "/BusiestReadTag");
+					.trackLatest(id.toString() + "_BusiestReadTag");
```

Comment:
The format that was used previously with the `/` is required for this to be read in status. Is there a need to change it?

Reply:
Forget about it. I misunderstood it by thinking it would be written to trace file...